### PR TITLE
Issues 06072018

### PR DESCRIPTION
**Pull request by mkosturkov**

### CODE REVIEW

File: src/Filters/CSS/FontSwap/Filter.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace Kibo\Phast\Filters\CSS\FontSwap;
+
+
+use Kibo\Phast\Services\ServiceFilter;
+use Kibo\Phast\ValueObjects\Resource;
+
+class Filter implements ServiceFilter {
+
+    const FONT_FACE_REGEXP = '/@font-face\s*\{/is';
```
Comment (apeschar) [inline src/Filters/CSS/FontSwap/Filter.php]: Remove the `s`!!!1

File: src/Filters/CSS/FontSwap/Filter.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    public function apply(Resource $resource, array $request) {
+        $css = $resource->getContent();
+        $filtered = preg_replace(self::FONT_FACE_REGEXP, '$0font-display: swap;', $css);
```
Comment (apeschar) [inline src/Filters/CSS/FontSwap/Filter.php]: Remove the ` ` :)

File: src/Filters/HTML/PhastScriptsCompiler/phast.js [hunk 3]
```diff
@@ -30,19 +30,32 @@ phast.wait = function (delay) {
...
+                return inspected;
+            }
+            inspected = inspected.nextSibling;
         }
+        return false;
+    }
+
+    statsElement = searchForStats(document.documentElement.nextSibling);
+    if (statsElement === false) {
+        statsElement = searchForStats(document.body.childNodes.item(0));
```
Comment (apeschar) [inline src/Filters/HTML/PhastScriptsCompiler/phast.js]: `firstChild`

**Comment by mkosturkov** [comment]
The requests from this unprecedented review have been met.

**Pull request merged**